File: src/favoritebusapp-client-reactvite/src/models/CtpDailyTimetable.ts
export type CtpDailyTimetable = {
  routeName: string;
  routeLongName: string;
  dayType: string;
  validFromDate: string;
  inStopName: string;
  outStopName: string;
  inStopTimes: string[];
  outStopTimes: string[];
};

File: src/FavoriteBusApp.Api/Timetables/CtpIntegration/Models/CtpDailyTimetable.cs
namespace FavoriteBusApp.Api.Timetables.CtpIntegration.Models;

public class CtpDailyTimetable
{
    public required string RouteName { get; set; }
    public required string RouteLongName { get; set; }
    public required string DayType { get; set; }
    public DateOnly? ValidFromDate { get; set; }
    public required string InStopName { get; set; }
    public required string OutStopName { get; set; }
    public List<TimeOnly> InStopTimes { get; set; } = [];
    public List<TimeOnly> OutStopTimes { get; set; } = [];
}
